diff --git a/src/daemon/configfile.c b/src/daemon/configfile.c
index d5f01e0..470a0f6 100644
--- a/src/daemon/configfile.c
+++ b/src/daemon/configfile.c
@@ -104,7 +104,7 @@ static cf_global_option_t cf_global_options[] = {
     {"BaseDir", NULL, 0, PKGLOCALSTATEDIR},
     {"PIDFile", NULL, 0, PIDFILE},
     {"Hostname", NULL, 0, NULL},
-    {"FQDNLookup", NULL, 0, "true"},
+    {"FQDNLookup", NULL, 0, "false"},
     {"Interval", NULL, 0, NULL},
     {"ReadThreads", NULL, 0, "5"},
     {"WriteThreads", NULL, 0, "5"},
